Sources/VLCEditToolbar.swift

```diff
@@ -10,10 +10,10 @@
  *****************************************************************************/
 
 protocol VLCEditToolbarDelegate: class {
-    func delete()
-    func createPlaylist()
-    func rename()
-    func share()
+    func delete(_ edittoolbar: VLCEditToolbar)
+    func addToPlaylist(_ edittoolbar: VLCEditToolbar)
+    func rename(_ edittoolbar: VLCEditToolbar)
+    func share(_ edittoolbar: VLCEditToolbar)
```

Comment:
I believe that we should rename the delegate methods to follow the naming convention a bit more.

See `UIApplicationDelegate`.

Reply:
suggestion? Again here I saw that even though a delegate was declared it didn't send the calling object and bettered that

Reply:
My suggestion is still `UIApplicationDelegate` so having something like:

`applicationDidEnterBackground(_:)`